src/modules/display.js

```diff
@@ -45,6 +56,34 @@ const commentPopup = (premiered, rating, type, runtime, image, title) => {
       }
     }
   };
+
+  const commentList = document.querySelector('.comment-list');
+  const commentCount = document.querySelector('#comment-count');
+  const eachComment = getComments(movieId);
+  eachComment.then((data) => {
+    commentCount.textContent = `(${data.length})`;
+    if (data.length > 0 || data.error !== "'item_id' not found.") {
+      data.forEach((comment) => {
+        const li = document.createElement('li');
+        li.innerHTML = `
+          <span id="comment-date">${comment.creation_date}</span>
+          <span id="comment-user">${comment.username}</span>
+          <span id="comment-msg">${comment.comment}</span>
+        `;
+        commentList.append(li);
+      });
+    }
+  });
+
+  const submitComment = document.getElementById('submit');
+  submitComment.addEventListener('click', (e) => {
+    e.preventDefault();
+    postComment(
+      movieId,
+      document.getElementById('user').value,
+      document.getElementById('comment').value,
```

Comment:
[**REQUIRED**] As part of the project requirements, the comment page should be updated upon user action (submit). As noticed on your website, the comment page updates only when the page is refreshed. Please try to fix this.